### ISSUE TITLE
show MKs presence punch-card / calendar of knesset events

### ISSUE THREAD

**Issue by OriHoch** [hasadna/Open-Knesset#184]
Show something like github punch-card:
https://github.com/OriHoch/Open-Knesset/graphs/punch-card
or just a calendar

we can show events like:
MKs presence
vote time
committee start / end time

etc..

this can allow to see if an MK was present during a vote but didn't vote or other such useful information

it partially depends on #182

**Comment by OriHoch** [comment]
should do design first, then split it into smaller more manageable development issues
